hack/release.toml

```
@@ -77,6 +77,10 @@ machine:
 ```
 
 Please note that on running cluster you will have to kill CoreDNS pods for this change to apply.
+
+The IP address used to forward DNS queries changed to be fixed `169.254.116.108`.
```

```suggestion
The IP address used to forward DNS queries has been changed to  fixed address `169.254.116.108`.
```

or probably better wording from Tim

---

thanks, fixed